src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -140,6 +140,7 @@ private function addAccessControlSection(ArrayNodeDefinition $rootNode)
                                 ->example('^/path to resource/')
                             ->end()
                             ->scalarNode('host')->defaultNull()->end()
+                            ->scalarNode('port')->defaultNull()->end()
```

Comment:
we probably don't want to deal with strings here, should be an `integerNode()` to me